tests/basic/longevity_test.go

```diff
@@ -743,6 +744,17 @@ func populateIntervals() {
 	triggerInterval[AsyncDRVolumeOnly][2] = 24 * baseInterval
 	triggerInterval[AsyncDRVolumeOnly][1] = 27 * baseInterval
 
+	triggerInterval[StorkApplicationBackup][10] = 1 * baseInterval
+	triggerInterval[StorkApplicationBackup][9] = 3 * baseInterval
+	triggerInterval[StorkApplicationBackup][8] = 6 * baseInterval
+	triggerInterval[StorkApplicationBackup][7] = 9 * baseInterval
+	triggerInterval[StorkApplicationBackup][6] = 12 * baseInterval
```

Comment:
here the base interval in 10 mins. is that fine?

Reply:
Yes, this has to run similar as above asyncDR test so should be fine.